lib/matplotlib/tests/test_axes.py

```diff
@@ -743,7 +743,7 @@ def test_pcolormesh_datetime_axis():
     fig.subplots_adjust(hspace=0.4, top=0.98, bottom=.15)
     base = datetime.datetime(2013, 1, 1)
     x = np.array([base + datetime.timedelta(days=d) for d in range(21)])
-    y = np.arange(21)
+    y = range(21)
```

Comment:
Actually `np.as{,any}array` handles `range` objects fine in Py3k.
But I do need to work on the test.